README.md

```diff
@@ -152,7 +155,11 @@ and
 
 ### Configuration
 
-To run Logprep, certain configurations have to be provided. 
+To run Logprep, certain configurations have to be provided. Because Logprep is designed to run in a
+containerized environment like Kubernetes these configurations can be provided via filesystem or
+http. By providing the configuration via http it is possible to control the configuration change via
+a flexible http api to make Logprep fast reacting to changes in your environment.
+
```

Comment:
```suggestion
To run Logprep, certain configurations have to be provided. Because Logprep is designed to run in a
containerized environment like Kubernetes, these configurations can be provided via the filesystem or
http. By providing the configuration via http, it is possible to control the configuration change via
a flexible http api. This enables Logprep to quickly adapt to changes in your environment.

```